scala-steward.conf

```diff
@@ -31,6 +31,11 @@ updates.ignore = [
   # See https://www.scala-lang.org/blog/2022/08/17/long-term-compatibility-plans.html#owners-of-commercial-projects
   {groupId = "org.scala-lang", artifactId = "scala3-library", version = {prefix = "3.4."}},
   {groupId = "org.scala-lang", artifactId = "scala3-library", version = {prefix = "3.5."}},
+
+  # Ignore updates to Apache Thrift and Scrooge Thrift.
+  # See https://trello.com/c/1rp2MVVA and https://github.com/guardian/content-api/issues/2903
+  {groupId = "org.apache.thrift", artifactId = "libthrift", version = {prefix = "0.20."}},
```

Comment:
I hadn’t realised we’d need to block libthrift as well, I expected it would just be scrooge. I haven’t done any testing though! Have you confirmed that this version update also breaks the build?

Reply:
You're right! I removed this.